base_layer/core/src/proof_of_work/mod.rs

```diff
@@ -25,6 +25,8 @@ mod difficulty;
 mod error;
 mod pow;
 
+pub mod lwma;
```

Comment:
Can we make this lwma_pow and LmwaPow like the blake one.

Reply:
I can make it lwma_diff?

This is not a pow, but only difficulty adjustment.

Reply:
Sorry, you are right. lwma_diff seems better